corehq/ex-submodules/couchexport/writers.py

```diff
@@ -582,7 +582,7 @@ def get_features(self, table, data):
                 result = row[geo_data_index].split(" ")
                 lat = result[0]
                 lng = result[1]
-            except ValueError:
+            except IndexError:
```

Comment:
The `IndexError` will occur if the `row[geo_data_index]` value is not correct (not in a form like `"lat lng ..."`), hence safe to ignore.